generators/app/templates/src/app/screens/Login/index.tsx

```diff
@@ -1,29 +1,37 @@
-import React, { Component } from 'react';
+import React, { useCallback } from 'react';
+import { useHistory } from 'react-router-dom';
+
+import { useDispatch } from '~app/context';
+import { actionCreators } from '~app/reducer';
```

Comment:
`as userActions` maybe?

Reply:
As I commented before, this reducer isn't limited to users. So I'd prefer it this way.